content/blog/2024/retrospective/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "2024 impact report: new team structure, new funding, and growth in our network"
+date: "2024-12-08"
+authors: ["Chris Holdgraf"]
+tags: [report]
+categories: [organization]
+featured: false
+draft: false
+---
+
+2024 has been a busy year for 2i2c, with many highs and lows, a lot of impact, and significant organizational change. As the year comes to an end, I want to reflect on the work we've done in 2024, and where we aim to go in 2025.
+
+## The main idea
+
+In 2024, 2i2c began an organizational transformation that I now call the "$1M to $2M budget jump". This is a point in an organization's lifecycle when your team has grown enough in size and complexity that you must change the ways that you organize. The informal ways that you worked as a small group don't suffice anymore, and you have to put more effort into aligning and coordinating everyone to ensure you have impact as a group.
```

Comment:
> that I now call the "$1M to $2M budget jump"
I don't have a suggested rewording for this, it just flows weirdly with the next paragraph (feel free to ignore this feedback)